src/test/java/com/odeyalo/sonata/connect/service/player/ChangePlayerVolumeTest.java

```diff
@@ -0,0 +1,109 @@
+package com.odeyalo.sonata.connect.service.player;
+
+import com.odeyalo.sonata.connect.entity.DevicesEntity;
+import com.odeyalo.sonata.connect.entity.PlayerStateEntity;
+import com.odeyalo.sonata.connect.exception.NoActiveDeviceException;
+import com.odeyalo.sonata.connect.model.CurrentPlayerState;
+import com.odeyalo.sonata.connect.model.Volume;
+import org.junit.jupiter.api.Test;
+import reactor.test.StepVerifier;
+import testing.asserts.DevicesAssert;
+import testing.faker.DeviceEntityFaker;
+import testing.faker.PlayerStateFaker;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static testing.factory.DefaultPlayerOperationsTestableBuilder.testableBuilder;
+
+class ChangePlayerVolumeTest extends DefaultPlayerOperationsTest {
+
+    @Test
+    void shouldReturnStateWithChangedVolume() {
+        final PlayerStateEntity playingPlayerState = PlayerStateFaker.create()
+                .user(existingUserEntity())
+                .get();
+
+        final DefaultPlayerOperations testable = testableBuilder().withState(playingPlayerState).build();
+
+        testable.changeVolume(EXISTING_USER, Volume.from(40))
+                .as(StepVerifier::create)
+                .assertNext(it -> assertThat(it.getVolume().asInt()).isEqualTo(40))
+                .verifyComplete();
+    }
+
+    @Test
+    void shouldSaveUpdatedState() {
+        final PlayerStateEntity playingPlayerState = PlayerStateFaker.create()
+                .user(existingUserEntity())
+                .get();
+
+        final DefaultPlayerOperations testable = testableBuilder().withState(playingPlayerState).build();
+
+        testable.changeVolume(EXISTING_USER, Volume.from(40))
+                .as(StepVerifier::create)
+                .expectNextCount(1)
+                .verifyComplete();
+
+        testable.currentState(EXISTING_USER)
+                .as(StepVerifier::create)
+                .assertNext(it -> assertThat(it.getVolume().asInt()).isEqualTo(40))
+                .verifyComplete();
+    }
+
+    // For single device - change volume
```

Comment:
Remove this comments

Reply:
Fixed via https://github.com/Project-Sonata/Sonata-Connect/pull/67/commits/53c36dab03833cf3b7fbb7e2a6e54e4d38c0feab